templates/web/base/report/new/duplicate_suggestions.html

```diff
@@ -0,0 +1,11 @@
+[% IF c.cobrand.suggest_duplicates %]
+<div id="js-duplicate-reports" class="duplicate-report-suggestions hidden">
+    <button class="duplicate-report-suggestions__close js-hide-duplicate-suggestions">[% loc('Close') %]</button>
+    <h2 class="form-section-heading">[% loc('Already been reported?') %]</h2>
+    <div class="form-section-description">
+        <p>[% loc('We’ve already sent these similar nearby reports to the council. You can track them to receive updates.') %]</p>
+    </div>
+    <ul class="item-list"></ul>
+    <button class="btn btn--block js-hide-duplicate-suggestions">[% loc('No, continue with my new report') %]</button>
```

Comment:
On mobile, this is immediately followed by the main navigation which looks a bit odd.

Reply:
Turns out this is also a problem _before_ the suggestion UI is even shown:

![screen shot 2019-02-20 at 11 19 20](https://user-images.githubusercontent.com/739624/53088382-7541d980-3501-11e9-9a15-e080ba982218.png)

Reply:
I’ve added a commit that fixes this for all map pages at mobile widths.